operator/internal/server/function.proto

```
@@ -0,0 +1,42 @@
+syntax = "proto3";
+
+package server;
```

Buf (1.64.0)

[error] Files with package "server" must be within a directory "server" relative to root but were in directory "operator/internal/server".

(PACKAGE_DIRECTORY_MATCH)